docs/scripting/functions/IsValidVehicle.md

```diff
@@ -0,0 +1,62 @@
+---
+id: IsValidVehicle
+title: IsValidVehicle
+description: Check if a vehicle is created.
+tags: []
+---
+
+:::note
+
+This function exists within the SA-MP server, but it has not been defined within the include files. To use this function, you must first declare the appropriate native at the top of your script:
```

Comment:
Link to the latest libraries and leave this note for the outdated libraries.

Reply:
Latest libraries: https://github.com/pawn-lang/samp-stdlib

Reply:
Could you elaborate on this? Not sure which libraries you are talking about. Also do you want the current text removed from the note?

Reply:
Something like this would be better:

```
:::note

This function is not present in the old libraries packaged with the SA-MP server however the [latest versions of the maintained libraries](https://github.com/pawn-lang/samp-stdlib) contain this and other formerly missing declarations.

:::
```

Reply:
Done.